radixdlt-java/src/main/java/com/radixdlt/client/core/atoms/particles/Particle.java

```diff
@@ -40,15 +41,12 @@ protected Particle() {
 		return addresses.stream().map(RadixAddress::getPublicKey).collect(Collectors.toSet());
 	}
 
-	public byte[] toDson() {
+	private byte[] toDson() {
 		return Serialize.getInstance().toDson(this, DsonOutput.Output.HASH);
 	}
 
-	public RadixHash getHash() {
+	@Override
+	public RadixHash hash() {
 		return RadixHash.of(toDson());
 	}
-
-	public EUID getHid() {
```

Comment:
This really should go back in and be final.

Reply:
Yup, was a bad idea.